perlmod/Fink/PkgVersion.pm

```diff
@@ -546,6 +546,9 @@ sub initialize {
 			} elsif ($config->param('Architecture') eq "x86_64" ) {
 				# paradoxically, no special library location is required for
 				# -64bit variants under x86_64 architecture
+			} elsif ($config->param('Architecture') eq arm64" ) {
```

Comment:
```suggestion
			} elsif ($config->param('Architecture') eq "arm64" ) {
```
Other than that, I just successfully completed a fresh bootstrap from this on 13.2. :)

Reply:
How was it passing tests like this ;) thanks for finding that